generated/nidcpower/nidcpower/_attributes.py

```diff
@@ -100,6 +101,38 @@ def __set__(self, session, value):
         return self._underlying_attribute.__set__(session, value.value)
 
 
+class AttributeEnumWithConverter(AttributeEnum):
+    '''Class for attributes that use enums internally but are exposed in nimi-python API as something else, thus need conversion.'''
```

Comment:
consider changing nimi-python for the module name.

Reply:
I have updated this in my latest commit.